docs/product-user-guide/src/main/asciidoc/guided-rule-templates-create-proc.adoc

```diff
@@ -1,19 +1,34 @@
-[id='guided_rule_templates_create_proc']
-= Creating Guided Rule Templates
+[id='guided-rule-templates-create-proc']
+= Creating guided rule templates
 
-You can create guided rule templates to define rule structure and data that can be used for other rules, instead of defining many different rule structures individually.
+You can use guided rule templates to define rule structures with placeholder values (template keys) that correspond to actual values defined in a data table. Guided rule templates are an efficient alternative to defining sets of many guided rules individually that use the same structure.
 
 .Procedure
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
 . Click *Create New Asset* -> *Guided Rule Template*.
-. Enter an informative *Guided Rule Template* name and select the appropriate *Package*.
+. Enter an informative *Guided Rule Template* name and select the appropriate *Package*. The package that you specify must be the same package where the required data objects have been assigned or will be assigned.
+. Click *Ok* to create the rule template.
 +
-The new Guided Rule Template is now created under the selected project.
-
-. Click the *Data Objects* tab and confirm that all data objects required for your rules are listed. If not, click *New item* to import objects from other packages, or **create new data objects(expand content from data-objects-create-proc.adoc) within the package.
-. After all data objects are in place, return to the *Editor* tab and use the buttons on the right of the editor window to add and define the *WHEN* (condition) and *THEN* (action) sections of the rule, based on the available data objects.
+The new guided rule template is now listed in the *Guided Rule Templates* panel of the *Project Explorer*.
```

Comment:
These documents target 7.x don't they? Users will normally be navigating a project's content with the "Library" screen that lists all assets in a project alphabetically without _grouping_ (by asset type). The "Project Explorer" is still available but only when an asset has been opened and the left-hand side "Dock" expanded. 

IDK therefore if you want to make such an explicit reference to the "Project Explorer"? Up to you.. as the value of mentioning it here depends on the overall visibility of the "Project Explorer" in the entire User Guide.

Reply:
@manstis , the reason I've written it this way is that when you create an asset, you're brought to the asset designer, with the collapsed Project Explorer to the left. You're not brought to the library list of assets. The fact that I'm referring to the Project Explorer when it is collapsed is just to give a bit more context of where the user is at in the moment after the asset creation.